### PR TITLE
Fixed polling timeout issue of start_db and start_node

### PR DESCRIPTION
**Pull request by cchen-vertica**
In k8s, the node state polling timeout is controlled by the annotation "vertica.com/restart-timeout". If it's not set, the polling timeout is set to 0 which will fail vclusterOps start_db and start_node. Then we will have many cluster/node restart failure events. This PR simply used vclusterOps default polling timeout if the annotation "vertica.com/restart-timeout" is not set.

### CODE REVIEW

File: pkg/vadmin/restart_node_vc.go [hunk 1]
```diff
@@ -62,7 +62,10 @@ func (v *VClusterOps) genStartNodeOptions(s *restartnode.Parms, certs *HTTPSCert
 	opts.UserName = su
 	opts.Password = &v.Password
 	opts.Nodes = s.RestartHosts
-	opts.StatePollingTimeout = v.VDB.GetRestartTimeout()
+	vdbTimeout := v.VDB.GetRestartTimeout()
```
Comment (spilchen) [inline pkg/vadmin/restart_node_vc.go:65]: Is this related to a specific change in vcluster? Or does this issue exist in 24.2.0. If it's an issue with 24.2.0, you may want to put the change into main first and pick it up in vnext via a merge.

Comment (chinhtranvan) [reply pkg/vadmin/restart_node_vc.go:65]: * For restart_node, I think the reason comes from my change: https://git.verticacorp.com/projects/VER/repos/server/pull-requests/20821/diff#platform/vcluster/vclusterops/start_node.go?f=149. I moved the default value to func (options *VStartNodesOptions) setDefaultValues(). Thank you for finding this error.

* For start_db, I think it exists in 24.2.0.

Comment (spilchen) [reply pkg/vadmin/restart_node_vc.go:65]: Because it is start_db, it probably should be in main and a changie entry added. The changie should only say you are fixing start_db. start_node is fine in prior releases.

**Comment by roypaulin** [comment]
I was this in `annotations.go`
```
// GetRestartTimeout returns the timeout to use for restart node or start db. If
// 0 is returned, this means to use the default.
func GetRestartTimeout(annotations map[string]string) int {
	return lookupIntAnnotation(annotations, RestartTimeoutAnnotation, 0 /* default value */)
}
```

Should we also set the default value here to 20min? Because admintools too gets the timeout value from this annotation

File: pkg/vadmin/restart_node_vc.go (hunk 1 above)
Comment (cchen-vertica) [reply pkg/vadmin/restart_node_vc.go:65]: start_db is fine in main. The change to make start_db fail is made last month in this PR https://git.verticacorp.com/projects/VER/repos/server/pull-requests/20685/diff#platform/vcluster/vclusterops/start_db.go?f=111

We don't need to put the change into main.

Comment (roypaulin) [reply pkg/vadmin/restart_node_vc.go:65]: I also suspect this is why the upgrade test fails in my PR

**Review by roypaulin — APPROVED** [review]
I am approving assuming start_db is fine in 24.2.0.
This will also fix my current PR.
